WordPressUI/FancyAlert/FancyAlertView.swift

```diff
@@ -177,4 +180,22 @@ open class FancyAlertView: UIView {
             buttonStackView.axis = .vertical
         }
     }
+
+    override open func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        // At the beginning of your implementation, call super to ensure that interface elements higher in the view hierarchy have an opportunity to adjust their layout first
+        super.traitCollectionDidChange(previousTraitCollection)
+        if previousTraitCollection?.preferredContentSizeCategory != traitCollection.preferredContentSizeCategory {
+            preferredContentSizeDidChange()
+        }
+    }
+
+    func preferredContentSizeDidChange() {
+        if #available(iOSApplicationExtension 11.0, *) {
```

Comment:
I am receiving an error here. Xcode wants to correct it to `if #available(iOS 11.0, *)` in my WCiOS project.

<img width="835" alt="screen shot 2019-01-11 at 9 31 58 am" src="https://user-images.githubusercontent.com/1062444/51043278-f06bb380-1583-11e9-84e0-5b755aad4ccd.png">

Reply:
Interesting... I'll take a look at it.
Thanks for checking that out 😁